lib/block-supports/align.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Align block support flag.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Registers the attributes block attribute for block types that support it.
+ *
+ * @param  array $block_type Block Type.
+ */
+function gutenberg_register_alignment_support( $block_type ) {
```

Comment:
Support flags can register block attributes.

Reply:
I would have expected this running on init to have solved the previous issue regarding the ServerSideRender components, but it still seems to be present.  On this branch im testing "Tag Cloud" block since we removed the attributes from block.json in favor of this:

1. Insert a "Tag Cloud" block.
2. Set alignment to "wide" width.
3. Verify we have an error with the block:
![Screen Shot 2020-08-04 at 4 53 16 PM](https://user-images.githubusercontent.com/28742426/89344121-63131700-d673-11ea-88ca-5f1926dcf634.png)
![Screen Shot 2020-08-04 at 4 53 22 PM](https://user-images.githubusercontent.com/28742426/89344134-673f3480-d673-11ea-8304-cf457158c7ca.png)

Reply:
yep good catch, I thought it would fix it but I'll have to debug further.

Reply:
@Addison-Stavlo Actually, I'm not able to reproduce this issue, everything works as intended for me.

Reply:
nevermind, I was testing with the plugin disabled 😬

Reply:
Fixed, for some reason, the hook priority need to be superior to 20